samples/features/sql-management-objects/src/CollectionSamples.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+using System.Diagnostics;
+using Microsoft.SqlServer.Management.Smo;
+
+namespace Microsoft.SqlServer.SmoSamples
+{
+    using System;

```

Comment:
Any particular reason you have the usings split up this? Just looks kind of weird

Reply:
visual studio decided to do it differently than what was already there